ansible/roles/edu-sharing/defaults/main.yml

```diff
@@ -54,3 +54,21 @@ oersi_port: "443"
 oersi_scheme: "https"
 oersi_pathprefix: "/resources/api-internal/search"
 oersi_index: "oer_data"
+
+# Default values for classifications keywords
+# define the classification-keyword interval in days = 1 means every day
+classification_keywords_update_interval: 15
+# give the path to the file where you want download the classification-keywords
+# must be public folder: example: /var/www/oer/
+classification_keywords_dir: "classification-keywords"
+#  classification keywords base dir
+classification_keywords_base_dir: "/var/www/{{classification_keywords_dir}}"
```

Comment:
Does this have to be configurable? Isn't this always fixed and derived from the other values? It has to be in the public-webserver dir, right? Then this can also be set directly in the task to avoid unnecessarily complex configuration.